common/beerocks/bwl/dwpal/sta_wlan_hal_dwpal.cpp

```diff
@@ -558,7 +558,7 @@ int sta_wlan_hal_dwpal::add_network()
     // Send command
     if (!dwpal_send_cmd(cmd, &reply)) {
         LOG(ERROR) << "ADD_NETWORK failed!";
-        return false;
+        return -1;
```

Comment:
```suggestion
        return INVALID_NETWORK_ID;
```
and define a `constexpr` in the appropriate place

Reply:
and change `sta_wlan_hal_dwpal::connect` (point of use) accordingly.